### ISSUE TITLE
Game crashes after 1 month

### ISSUE THREAD

**Issue by sobisonator** [sobisonator/imp19c#35]
Crash detected when using mod as of Commit 101

It looks like it may be due to a vanilla event to do with Carthage trying to fire after one month. Alternatively it may be due to the character culture purge event because of the error message `[14:12:44][charactertriggerimpl.cpp:1114]: has_same_culture_as : has_same_culture_as target was null, at:  file: events/character_inequality.txt line: 22`

The last few lines of the error log indicate the following:
```
[14:12:44][jomini_script_system.cpp:176]: Script system error!
  Error: Event target link 'c' returned an unset scope
  Script location: file: events/dhe/dhe_carthage.txt line: 889
[14:12:44][jomini_script_system.cpp:176]: Script system error!
  Error: context_trigger trigger [ Failed context switch ]
  Script location: file: events/dhe/dhe_carthage.txt line: 889
[14:12:44][jomini_script_system.cpp:176]: Script system error!
  Error: Failed to scope to country via tag 'CAR'
  Script location: file:  file: events/dhe/dhe_carthage.txt line: 1182; owns_mediterranean_coast_as_carthage_conquer_trigger line: 3
[14:12:44][jomini_script_system.cpp:176]: Script system error!
  Error: Event target link 'c' returned an unset scope
  Script location: file:  file: events/dhe/dhe_carthage.txt line: 1182; owns_mediterranean_coast_as_carthage_conquer_trigger line: 3
[14:12:44][provincetriggerimpl.cpp:1862]: owned_or_subject_owned : target was null, at:  file:  file: events/dhe/dhe_carthage.txt line: 1182; owns_mediterranean_coast_as_carthage_conquer_trigger line: 3
[14:12:44][jomini_script_system.cpp:176]: Script system error!
  Error: Invalid country (MER) found for link 'current_ruler', country is not real or valid
  Script location: file: events/character_inequality.txt line: 22
[14:12:44][jomini_script_system.cpp:176]: Script system error!
  Error: Event target link 'current_ruler' returned an unset scope
  Script location: file: events/character_inequality.txt line: 22
[14:12:44][charactertriggerimpl.cpp:1114]: has_same_culture_as : has_same_culture_as target was null, at:  file: events/character_inequality.txt line: 22
```

**Comment by sobisonator** [comment]
Turns out the crash was due to neither. Fixed the character-to-employer culture scoping as well as transplanting a blank Carthage event, but the crash persists.

Error log of no help.

**Comment by sobisonator** [comment]
Cause identified: religions need omens and deities assigned and created.

Going to work on this now.

**Comment by sobisonator** [comment]
Resolved. Now just need to add flavour to religions, marking this as closed.

**Comment by sobisonator** [comment]
Re-opened. Day 4 crash was resolved, end of month 2 crash still occurring.

**Comment by sobisonator** [comment]
Crash resolved since patch 1.5
